ribs/archives/_cvt_archive.py

```diff
@@ -124,7 +125,8 @@ def __init__(self,
                  chunk_size=None,
```

Comment:
Can you rearrange the arguments to be: custom_centroids, centroid_method, samples, k_means_kwargs, use_kd_tree, ckdtree_kwargs, chunk_size? I think this makes more sense because it has the centroid args first, then the kdtree / nearest neighbor args.

Also rearrange this order in the docstring above, and also add documentation for `centroid_method` in the docstring. For the centroid_method docstring, make sure to list all allowed methods.